.github/mergify.yml

```diff
@@ -45,3 +43,71 @@ pull_request_rules:
           This pull request has merge conflicts that must be resolved before it can be merged. @{{author}} please update it 🙏.
 
           Try `@mergify update` or update manually.
+
+  - name: Welcome new contributor
+    conditions:
+      - author!=Mergify
+    actions:
+      comment:
+        message: |
+          Thanks for the contribution!
+          I have applied any labels matching special text in your PR Changelog.
+
+          Please review the labels and make any necessary changes.
+
+  - name: label on New Feature
+    conditions:
+      - body~=New Feature
+    actions:
+      label:
+        add:
+          - pr-feature
+  - name: label on Bug Fix
+    conditions:
+      - body~=Bug Fix
+    actions:
+      label:
+        add:
+          - pr-bugfix
+  - name: label on Improvement
+    conditions:
+      - body~=Improvement
+    actions:
+      label:
+        add:
+          - pr-improvement
+  - name: label on Improvement
```

Comment:
It seems this label will never be set since the above:
```
  conditions:
      - body~=Improvement
  ```

Reply:
They will be applied at the same time. Maybe we can adopt more strict rules for the behavior.

Reply:
![image](https://user-images.githubusercontent.com/5351546/154055418-9ec46aaa-825b-45ff-a78c-8118908425c0.png)

Reply:
Fixed by checking `- Improvement` instead of `Improvement`